src/xrEngine/xr_ioc_cmd.cpp

```diff
@@ -1,4 +1,7 @@
 #include "stdafx.h"
+
+#include <algorithm>
```

Comment:
Oh, sorry, the last thing: unnecessary include, please remove it.

Reply:
Yep, my fault